### PR TITLE
Update README.md to describe the repo.

### PR DESCRIPTION
**Pull request by zrideaux**
Added a brief description of the repository as suggested in the walkthrough. Also kept the old description wrapped in a code block.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# STEP Starter Projects (zrideaux)
+
+This repo contains my portfolio and STEP projects.
+
+```md
```
Comment (saucepwn) [inline README.md:5]: Are the three backticks intentional? They cause the paragraph below to lose its markdown formatting.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,12 +7,30 @@
...
+        <p>I am a rising sophomore at UC Irvine who is currently serving as a STEP Intern at Google! =)</p>
+        <img src="/images/cat.png">
+        <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>
+      </div>
+
+      <div id="work" class="tab-content">
+        <h1>My Work</h1>
+        <p>Test</p>
```
Comment (saucepwn) [inline portfolio/src/main/webapp/index.html:32]: Should there be something more than a placeholder here?

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+  text-align: center;
+  width: 50px;
+}
+
+#initial-z {
+  color: #FFFFFF;
+}
+
+#initial-r {
+  color: rgb(238, 37, 121);
```
Comment (saucepwn) [inline portfolio/src/main/webapp/style.css:27]: As a general rule color specs should be consistent; the entire file should either use hex, rgb, or hsv, unless there's a specific reason to use one (such as using rgba to control the alpha channel)

File: README.md (hunk 1 above)
Comment (zrideaux) [reply README.md:5]: They were, I wanted to preserve the original text at the time, although the whole thing can be omitted now. 🙂

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (zrideaux) [reply portfolio/src/main/webapp/index.html:32]: Will expand soon! For the time being I just wanted something there to test the implementation of the tab buttons.

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (zrideaux) [reply portfolio/src/main/webapp/style.css:27]: Noted, thank you!

**Pull request merged**